uber/utils.py

```diff
@@ -67,10 +73,12 @@ def check_csrf(csrf_token):
     """
     if csrf_token is None:
         csrf_token = cherrypy.request.headers.get('CSRF-Token')
-    assert csrf_token, 'CSRF token missing'
+    if __debug__:
```

Comment:
kill if __debug__: here (which you did because you're physically sitting next to me), I totally get why you did this :)  but in fact, we do actually want to always do this check.  we were just using 'assert' because it's convenient, when in fact we always want to do this check even if __debug__ is False